quinn-proto/src/endpoint.rs

```diff
@@ -486,19 +513,26 @@ impl Endpoint {
             }
         };
 
-        Some(DatagramEvent::NewConnection(Incoming {
-            addresses,
-            ecn,
-            packet: InitialPacket {
-                header,
-                header_data: packet.header_data,
-                payload: packet.payload,
+        let incoming_idx = self.incoming_buffers.insert(Default::default());
+        self.index
+            .insert_initial_incoming(orig_dst_cid, incoming_idx);
```

Comment:
How do we limit the number of these? Buffer size limits will not be an effective defense if the number of buffers is unbounded.

Reply:
See comment below.